dwellingly/src/App.js

```diff
@@ -90,6 +91,7 @@ export class App extends React.Component {
               <PrivateRoute exact path='/' component={Dashboard} />
               <Route exact path='/login' component={LoginForm} />
               <PrivateRoute exact path='/dashboard' component={Dashboard} />
+              <PrivateRoute exact path='/add-property' component={AddProperty} />
```

Comment:
We have the `/add/property` PrivateRoute in the list below from the left nav, so we won't need this one